scripts/ableplayer.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'ableplayer';
```

Comment:
Surely you have to declare also an export `AblePlayer`? otherwise I think there will be a type error that the module ableplayer does hot define an export `AblePlayer`

Reply:
hmm so far there isn't a type error for AblePlayer 🤔 but yes now that you've mentioned I should include AblePlayer in here

Reply:
I mean a type error in OLC when you try to import it. You can probably test it with yarn link, and another option is using the @types/ directory in OLC (even just temporarily to ensure that the types work from the point of view of the user module)

Reply:
I know - I just tried it on Client (with yarn link) and there's no error.. I was also expecting it to but for some reason it did not complain about AblePlayer (maybe because VideoWidget is not `tsx`?)

Reply:
okok I'm not giving up yet... let me try one last thing...

Reply:
@hedgepigdaniel  BEHOLD!! GOT IT IN THE END! ☝️
Turns out, only export if it's from the tsx files, and `declare` if it's from an external non-tsx module. No need for `export` if I do `declare` but I *did* add one for the constructor 😁

Reply:
I think I earned the ✅ , give me a ✅ (🤞)

Reply:
Haha, yeah I haven't really worked out what the `declare module` thing is for - I think you can also skip that and have `export` statements - not really sure what the difference is